web/src/main/angular/src/app/core/components/server-map/class/server-map-data.class.ts

```diff
@@ -16,6 +16,8 @@ export interface IShortNodeInfo {
     hasAlert?: boolean;
     slowCount?: number;
     histogram?: IResponseTime | IResponseMilliSecondTime;
+    responseStatistics?: IResponseStatistics;
+    agentResponseStatistics:  { [key: string]: IResponseStatistics};
```

Comment:
Take it as an optional property.
```suggestion
    agentResponseStatistics?:  { [key: string]: IResponseStatistics};
```